nano/lib/rpcconfig.cpp

```diff
@@ -44,44 +47,93 @@ nano::error nano::rpc_config::serialize_json (nano::jsonconfig & json) const
 	json.put ("port", port);
 	json.put ("enable_control", enable_control);
 	json.put ("max_json_depth", max_json_depth);
-	json.put ("enable_sign_hash", enable_sign_hash);
 	json.put ("max_request_size", max_request_size);
-	json.put ("max_work_generate_difficulty", nano::to_string_hex (max_work_generate_difficulty));
+	json.put ("io_threads", io_threads);
+	json.put ("ipc_port", ipc_port);
+	json.put ("ipc_path", ipc_path);
```

Comment:
Is this used? I don't think the external RPC process supports using domain sockets yet.

Reply:
Nope was an artifact when I was also migrating the values in the ipc config. Removed